app/src/main/java/com/lorenzovainigli/foodexpirationdates/ui/theme/Color.kt

```diff
@@ -38,7 +38,7 @@ val md_theme_dark_onPrimary = Color(0xFF4F2500)
 val md_theme_dark_primaryContainer = Color(0xFF703800)
 val md_theme_dark_onPrimaryContainer = Color(0xFFFFDCC5)
 val md_theme_dark_secondary = Color(0xFFE4BFA7)
-val md_theme_dark_onSecondary = Color(0xFF422B1A)
+val md_theme_dark_onSecondary = Color(0xFF422B1A)//
```

Comment:
Has this been included in the commit by mistake?

Reply:
ohh yeah sorry it was by mistake